packages/babel-compiler/babel-compiler.js

```diff
@@ -145,7 +146,51 @@ BCp.processOneFileForTarget = function (inputFile, source) {
 
     try {
       var result = profile('Babel.compile', function () {
-        return Babel.compile(source, babelOptions, cacheOptions);
+        let compilation;
+        try {
+          const packagesSkipSwc = [];
+          const fileSkipSwc = ['webapp_server.js']; // top level await
```

Comment:
This was skipped from SWC compilation due to top-level await: [link](https://github.com/meteor/meteor/blob/devel/packages/webapp/webapp_server.js#L1558).

Not sure yet if we can ignore or change this, but skipping one file doesn't seem too impactful. Maybe I could isolate this await and related functions in a smaller file, though it's not really important.